x-pack/plugins/security/server/authentication/unauthorized_page.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// @ts-expect-error no definitions in component folder
+import { EuiButton } from '@elastic/eui/lib/components/button';
+// @ts-expect-error no definitions in component folder
+import { EuiEmptyPrompt } from '@elastic/eui/lib/components/empty_prompt';
+// @ts-expect-error no definitions in component folder
+import { icon as EuiIconAlert } from '@elastic/eui/lib/components/icon/assets/alert';
+// @ts-expect-error no definitions in component folder
+import { appendIconComponentCache } from '@elastic/eui/lib/components/icon/icon';
+// @ts-expect-error no definitions in component folder
+import { EuiPage, EuiPageBody, EuiPageContent } from '@elastic/eui/lib/components/page';
+import React from 'react';
+import { renderToStaticMarkup } from 'react-dom/server';
+
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage, I18nProvider } from '@kbn/i18n/react';
+import * as UiSharedDeps from '@kbn/ui-shared-deps';
+import type { IBasePath } from 'src/core/server';
+
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { Fonts } from '../../../../../src/core/server/rendering/views/fonts';
+
+// Preload the alert icon used by `EuiEmptyPrompt` to ensure that it's loaded
+// in advance the first time this page is rendered server-side. If not, the
+// icon svg wouldn't contain any paths the first time the page was rendered.
+appendIconComponentCache({
+  alert: EuiIconAlert,
+});
+
+interface Props {
+  originalURL: string;
+  buildNumber: number;
+  basePath: IBasePath;
+}
+
+export function UnauthorizedPage({ basePath, originalURL, buildNumber }: Props) {
```

Comment:
Nit: I feel like calling this `UnauthorizedPage` is a _bit_ of a misnomer. I know the HTTP status code is "401 Unauthorized", but what we're really saying here is we were not able to authenticate the user at all, and they need to try to log in again.

How do you feel about something like `UnauthenticatedPage`, `NotAuthenticatedPage`, or `FailedAuthenticationPage`?

Reply:
++ I struggled with the naming too. I 100% understand why you went with `Unauthorized`, though!

`UnauthenticatedPage` gets my vote for an alternate name

Reply:
Yeah, took the `to be technically correct is the best way to be correct` path, but definitely agree - let's use `UnauthenticatedPage` then, thanks!